123_state_tasks.R

```diff
@@ -60,8 +60,9 @@ do_state_tasks <- function(oldest_active_sites, ...) {
     as_promises = TRUE)
 
   # Build the tasks
-  obs_tallies <- scmake('obs_tallies_promise', remake_file='123_state_tasks.yml')
-  scmake('timeseries_plots.yml_promise', remake_file='123_state_tasks.yml')
+  loop_tasks(task_plan, '123_state_tasks.yml', num_tries = 100)
+
+  obs_tallies <- remake::fetch('obs_tallies_promise', remake_file='123_state_tasks.yml')
```

Comment:
I haven't yet changed this in the course, but note that there's now `scipiper::fetch()` as of that last PR you merged in **scipiper**